glide.yaml

```diff
@@ -4,7 +4,7 @@ import:
 testImport:
 - package: github.com/icrowley/fake
 - package: github.com/stretchr/testify
-  version: ^1.1.3
+  version: ^1.2.1
```

Comment:
I need to bump this version. I was seeing error like below with 1.1.3. 1.2.1 works well
```
╰─± go test .
# github.com/leebenson/conform/vendor/github.com/stretchr/testify/assert
vendor/github.com/stretchr/testify/assert/assertions.go:1253:25: unknown field 'DisablePointerAddresses' in struct literal of type spew.ConfigState
vendor/github.com/stretchr/testify/assert/assertions.go:1254:19: unknown field 'DisableCapacities' in struct literal of type spew.ConfigState
FAIL    github.com/leebenson/conform [build failed]
```